### ISSUE TITLE
QA: Configure the name by branch

### ISSUE THREAD

**Issue by EinfachHans** [pustovitDmytro/semantic-release-heroku#50]
This is more of a Support Request 😃 

I have two branches **master** and **staging** and two heroku projects.
How can i dynamically set the correct name in the plugin's config depending on the current branch?

**Comment by pustovitDmytro** [comment]
Hi, thank you for using `semantic-release-heroku`. You can achieve such behaviour using semantic-release keyword `branch`

Simple example:
```
[ 
    "semantic-release-heroku", {
        "branch": "staging",
        "name": "staging-name"
    } 
],
[ 
    "semantic-release-heroku", {
        "branch": "master",
        "name": "production-name"
    } 
]
```

Please reply here if such a configuration works for you.

**Comment by EinfachHans** [comment]
@pustovitDmytro That's amazing, thank you! 😊 Is that a default keyword that i can use out of the box for every plugin to define if it should run or not?

**Comment by pustovitDmytro** [comment]
Well, as I believed earlier there was such filter at semantic-release. But unfortunately can't find it now at the documentation. So I assume this approach won't work with the latest semantic-release. 

I consider this to be an important feature, so now I'm going to add this to plugin directly.

Sorry, for the "false way"

**Comment by EinfachHans** [comment]
Ahh damn that's sad, would be great for other plugins for me as well 😃 

Yeah i searched a lot for something like "semantic release plugin configuration by branch" but found nearly nothing like i'm the only one with this question 😃 

If you can add this to the plugin directly this would be great!

**Comment by pustovitDmytro** [comment]
@EinfachHans  no, still work in progress. I'll leave usage notes here when the new version is ready.

**Comment by EinfachHans** [comment]
Amazing!! Thank you! 😊
